src/main/scala/beam/router/skim/core/ODSkimmer.scala

```diff
@@ -373,8 +381,8 @@ object ODSkimmer extends LazyLogging {
         cost = row("cost").toDouble,
         energy = Option(row("energy")).map(_.toDouble).getOrElse(0.0),
         level4CavTravelTimeScalingFactor = row.get("level4CavTravelTimeScalingFactor").map(_.toDouble).getOrElse(1.0),
-        observations = row("observations").toInt,
-        iterations = row("iterations").toInt
+        observations = toIntOrDefaultIfNull(row("observations"), 0),
```

Comment:
Maybe we could use `org.apache.commons.lang3.math.NumberUtils#toInt(java.lang.String, int)` from apache commons-lang?